client/views/windows/settings.tpl

```diff
@@ -55,6 +55,13 @@
 				Enable autocomplete
 			</label>
 		</div>
+		<div class="col-sm-12">
+			<label class="opt">
+				<label for="nickPostfix">Nick autocomplete postfix (e.g. <code>, </code>)</label>
+				<input type="text" id="nickPostfix" name="nickPostfix" class="input" placeholder="Nick autocomplete postfix">
```

Comment:
Could you add `class="sr-only"` to the `label`, and ` (e.g. ", ")` to the `placeholder`, to match the custom highlights fields please?
It's not an idea look, I know, but to keep things consistent for now.

Reply:
Did that.